nuget/OpenTelemetry.AutoInstrumentation.nuspec

```diff
@@ -13,44 +13,44 @@
-        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.13" />
-        <dependency id="OpenTelemetry.Instrumentation.Quartz" version="1.0.0-alpha.1" />
-        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.1.0-rc.1" />
-        <dependency id="OpenTelemetry.Instrumentation.Process" version="0.5.0-beta.1" />
-        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.13" />
-        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.8" />
-        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.13" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.4.0" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.4.0" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.14" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.14" />
+        <dependency id="OpenTelemetry.Instrumentation.Process" version="0.5.0-beta.2" />
+        <dependency id="OpenTelemetry.Instrumentation.Quartz" version="1.0.0-alpha.2" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.1.0-rc.2" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.14" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.9" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.14" />
         <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs" version="1.4.0-rc.4" />
-        <dependency id="OpenTelemetry.Instrumentation.AspNetCore" version="1.0.0-rc9.13" />
-        <dependency id="OpenTelemetry.Instrumentation.EntityFrameworkCore" version="1.0.0-beta.4" />
-        <dependency id="OpenTelemetry.Instrumentation.MySqlData" version="1.0.0-beta.5" />
-        <dependency id="OpenTelemetry.Instrumentation.StackExchangeRedis" version="1.0.0-rc9.7" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNetCore" version="1.0.0-rc9.14" />
+        <dependency id="OpenTelemetry.Instrumentation.EntityFrameworkCore" version="1.0.0-beta.5" />
+        <dependency id="OpenTelemetry.Instrumentation.MySqlData" version="1.0.0-beta.6" />
+        <dependency id="OpenTelemetry.Instrumentation.StackExchangeRedis" version="1.0.0-rc9.8" />
       </group>
       <group targetFramework=".NETFramework4.6.2">
-        <dependency id="OpenTelemetry" version="1.4.0-rc.4" />
-        <dependency id="OpenTelemetry.Api" version="1.4.0-rc.4" />
-        <dependency id="OpenTelemetry.Exporter.Console" version="1.4.0-rc.4" />
+        <dependency id="OpenTelemetry" version="1.4.0" />
+        <dependency id="OpenTelemetry.Api" version="1.4.0" />
+        <dependency id="OpenTelemetry.Exporter.Console" version="1.4.0" />
         <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.4.0-rc.4" />
```

Comment:
bump to 1.4.0

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2268/commits/f26ce55f7f40903b91d93b62ea948482efe436bf